### PR TITLE
OHFJIRA-68 - No routes are load in CamelContext

### PR DESCRIPTION
**Pull request by openhubframework**
Problem:
Loading all routes is in web child context, but camel context starts in root application context. 

Solution:
Move route loading (class CamelRoutesConfig) and base security config (GlobalSecurityConfig and WebSecurityConfig) from child web context into root context. Remove child context. Everything now is in root context.

JIRA:
https://openhubframework.atlassian.net/browse/OHFJIRA-68

### CODE REVIEW

**Comment by hanusto** [comment]
It is ready for merge? I know that you plan to update this PR.

**Comment by hanusto** [comment]
This branch is out-of-date with the base branch, please, rebase that on to develop.

**Comment by hanusto** [comment]
This branch has conflict with WebContextConfig. Actually it is not possible to be merged. Can you please resolve it?

**Comment by openhubframework** [comment]
All conflicts are resolved.

**Review by hanusto — APPROVED** [review]
Please, can you test extensions and we have to update documentation.

File: web-admin/src/main/java/org/openhubframework/openhub/web/config/WebServiceConfigurer.java [hunk 1]
```diff
@@ -28,17 +12,20 @@
 import org.openhubframework.openhub.api.route.RouteConstants;
 import org.openhubframework.openhub.core.common.ws.ErrorCodeAwareWebServiceMessageReceiverHandlerAdapter;
 
-
 /**
- * Web configurer which is responsible for configuration of web runtime layer of OpenHub, for example servlet
- * registration of camel connectors.
+ * Web configurer which is responsible for servlet registration of camel connectors.
```
Comment (pjuza) [inline web-admin/src/main/java/org/openhubframework/openhub/web/config/WebServiceConfigurer.java:16]: Wrong JavaDoc comment ...

**Review by pjuza — APPROVED** [review]

File: web-admin/src/main/java/org/openhubframework/openhub/web/config/WebServiceConfigurer.java (hunk 1 above)
Comment (hanusto) [reply web-admin/src/main/java/org/openhubframework/openhub/web/config/WebServiceConfigurer.java:16]: I will fix that after rebase on the top of develop.

**Pull request merged**